input/examples/practitioner-chau-fryer.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Practitioner xmlns="http://hl7.org/fhir">
+    <id value="chau-fryer"/>
+    <meta>
+        <profile value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-practitioner"/>
+    </meta>
+    <name>
```

Comment:
add name use, make last name not all caps, add a prefix

Reply:
Done